lib/bapig.rb

```diff
@@ -11,6 +11,7 @@ def initialize(email, password)
     puts response.code
     raise "invalid email/password " if response.code == 401
     @auth_token = response["auth_token"]
+    @auth_user = response["user"]
```

Comment:
I'm not clear on what the point of this variable is. It is not harming anything but I want you to be mindful of creating class variables if you do not use them.

Reply:
@wolffles ^

Reply:
oh yeah I was trying to create a variable to get emails of the user but i had some trouble working with it, and i went the easier route and just forgot to remove it